### ISSUE TITLE
[regtool] Maybe qe signals should be per-register, rather than per-field

### ISSUE THREAD

**Issue by rswarbrick** [lowRISC/opentitan#5429]
At the moment, a register with hwqe will generate one "qe" signal per field. This is a bit silly, because these signals are all equal. See https://github.com/lowRISC/opentitan/pull/5426#discussion_r585663204 for a lint-fix needed because of it. Maybe it would make more sense just to generate a "qe" for the whole register.

@eunchan: Do you have an opinion on what would be better?

**Comment by eunchan** [comment]
I agree. As we mandate the host writes full-word (or at least cover the valid fields), having one `qe` per register makes sense.

**Comment by msfschaffner** [comment]
We can look into cleaning this up, but it may have quite some impact on existing IPs. Hence I am marking this as a P3.

**Comment by msfschaffner** [comment]
Due to potential impact on existing IPs, I am moving this into the backlog for now.

**Comment by rswarbrick** [comment]
Ah, this was a bad idea! @alees24 and I chatted yesterday and I think I now understand a bit more about what a `qe` signal should do and how partial register writes might be handled in future.
